### PR TITLE
Add descriptions for missing commands in lastseen & certauth & route_replies

### PR DESCRIPTION
**Pull request by Mikaela**
I checked all modules that I had loaded and only these three in addition to perform (https://github.com/znc/znc/pull/1102) and cert (https://github.com/znc/znc/pull/1103) are missing descriptions in commands.

### CODE REVIEW

**Comment by Mikaela** [comment]
I am not sure if I should squash these commits, but I assume no, because they touch different modules.

In case I should squash, I am waiting for opinion from someone with repo-access.

File: modules/route_replies.cpp [hunk 1]
```diff
@@ -218,7 +218,7 @@ class CRouteRepliesMod : public CModule
 
 		AddHelpCommand();
 		AddCommand("Silent", static_cast<CModCommand::ModCmdFunc>(&CRouteRepliesMod::SilentCommand),
-			"[yes|no]");
+			"[yes|no]"), "Decides whether to show the timeout messages or not";
```
Comment (Zarthus) [inline modules/route_replies.cpp]: Does this compile? It seems doubtful with the missing brace.

Comment (Mikaela) [inline modules/route_replies.cpp]: Apparently it did, but it's still wrong.

Comment (Mikaela) [inline modules/route_replies.cpp]: Fixed.

**Comment by DarthGandalf** [comment]
5 commits for this is certainly too much. During `git rebase -i` you can select what exactly happens with which exact commit. Reordering and `fixup` looks right here.
Though 1 commit would be fine too.

**Comment by Mikaela** [comment]
Done

**Pull request merged**